src/main.ts

```diff
@@ -37,14 +37,14 @@ global.exportCsv = exportCsv
  * 指定した銘柄の財務数字や指標を取得します。
  *
  * @param {"6501"} ticker 銘柄コード
- * @param {"2017Q4"} period 会計期間 (例: 四半期 '2017Q4', 日付 '2020-09-06')
+ * @param {"2017Q4"} intent 会計期間 (例: 四半期 '2017Q4', 日付 '2020-09-06')、企業情報を取得する場合は`COMPANY'を指定してください
```

Comment:
ここの文言をどうするか 🤔

Reply:
自分が書くなら以下のようにすると思いますが、今の文言でもいいと思います。

```suggestion
 * @param {"2017Q4"} intent 会計期間または識別子 (例: 四半期 '2017Q4', 日付 '2020-09-06', 企業情報 'COMPANY')
```